.travis.yml

```diff
@@ -61,6 +61,14 @@ jobs:
         - travis_retry vale sync # pull down VRH rules package
         - chmod +x ./scripts/check-with-vale.sh
         - travis_retry ./scripts/check-with-vale.sh $TRAVIS_PULL_REQUEST $TRAVIS_PULL_REQUEST_SHA
+    - stage: check-yaml
+      if: type IN (pull_request)
+      name: "Run yamllint against PR YAML files"
+      before_script:
+        - travis_retry sudo apt-get install yamllint
+      script:
+        - chmod +x ./scripts/lint-yaml.sh
+        - ./scripts/lint-yaml.sh
```

Comment:
Not sure but would it be an option to run this action on push instead: https://github.com/marketplace/actions/yaml-lint

Reply:
Yes! For some reason, OCP doesn't use actions. I have no idea why tbh.

Reply:
Huh. Well, you could add the action to your fork and run the action there. At least you can evaluate the option but if they're using Travis for CI then it might be a hard sell. People get finicky about their CI tooling.